docs/sources/next/using-k6-browser/migrating-to-k6-v0-52.md

```diff
@@ -0,0 +1,309 @@
+---
+title: 'Migrating browser scripts to k6 v0.52'
+description: 'A migration guide to ease the process of transitioning to the new non-experimental k6 browser module version bundled with k6 v0.52'
+weight: 04
+---
+
+# Migrating browser scripts to k6 v0.52
+
+This guide outlines the key changes you will need to make when moving your existing k6 browser test scripts to the _latest_ non-experimental [k6 browser module](https://grafana.com/docs/k6/<K6_VERSION>/javascript-api/k6-experimental/browser) version bundled with [k6 0.52](https://github.com/grafana/k6/releases/tag/v0.52.0).
+
+## Key changes
+
+In the latest release of k6, we have introduced the non-experimental k6 browser module under the import `k6/browser`. Migrating to this is a breaking change that will affect _all scripts_ that use the current experimental k6 browser module. The breaking changes are:
```

Comment:
Also, here: "non-experimental." Keeping "experimental" at the end of the sentence is good, though (to separate).